packages/malloy/src/dialect/trino/dialect_functions.ts

```diff
@@ -62,9 +145,117 @@ const string_agg_distinct: OverloadedDefinitionBlueprint = {
   },
 };
 
+const variance: DefinitionBlueprint = {
+  takes: {'value': {dimension: 'number'}},
+  returns: {measure: 'number'},
+  impl: {function: 'VARIANCE'},
+};
+
+// Scalar functions
+
+const bitwise_and: DefinitionBlueprint = {
+  takes: {'val1': 'number', 'val2': 'number'},
+  returns: 'number',
+  impl: {
+    sql: 'BITWISE_AND(${val1}, ${val2})',
+  },
+};
+
+const bitwise_or: DefinitionBlueprint = {
+  takes: {'val1': 'number', 'val2': 'number'},
+  returns: 'number',
+  impl: {
+    sql: 'BITWISE_OR(${val1}, ${val2})',
+  },
+};
+
+const date_format: DefinitionBlueprint = {
+  takes: {'ts_val': 'timestamp', 'format': 'string'},
+  returns: 'string',
+  impl: {
+    sql: 'DATE_FORMAT(${ts_val}, ${format})',
+  },
+};
+
+const date_parse: DefinitionBlueprint = {
+  takes: {'ts_string': 'string', 'format': 'string'},
+  returns: 'timestamp',
+  impl: {
+    sql: 'DATE_PARSE(${ts_string}, ${format})',
+  },
+};
+
+const from_unixtime: DefinitionBlueprint = {
+  takes: {'unixtime': 'number'},
+  returns: 'timestamp',
+  impl: {function: 'FROM_UNIXTIME'},
+};
+
+const json_extract_scalar: DefinitionBlueprint = {
+  takes: {'json_val': 'string', 'json_path': 'string'},
+  returns: 'string',
+  impl: {sql: 'JSON_EXTRACT_SCALAR(${json_val}, ${json_path})'},
+};
+
+const regexp_like: DefinitionBlueprint = {
+  takes: {'str': 'string', 'pattern': 'string'},
+  returns: 'boolean',
+  impl: {sql: 'REGEXP_LIKE(${str}, ${pattern})'},
+};
+
+const regexp_replace: OverloadedDefinitionBlueprint = {
+  remove_matches: {
+    takes: {'input_val': 'string', 'regexp_pattern': 'string'},
+    returns: 'string',
+    impl: {
+      sql: 'REGEXP_REPLACE(${input_val}, ${regexp_pattern})',
+    },
+  },
+
+  replace_matches: {
+    takes: {
+      'input_val': 'string',
+      'regexp_pattern': 'string',
+      'replace_pattern': 'string',
```

Comment:
I don't think this should be called `replace_pattern`, as that suggests it's a regular expression. In the presto docs, it's a string called `replacement`.